Tensile/Common.py

```diff
@@ -63,7 +64,7 @@
 globalParameters["CodeFromFiles"] = True          # if False byte arrays will be generated during Benchmarking phase as before
 globalParameters["SortProblems"] = False          # sort problems by size; else use order in YAML file
 globalParameters["PinClocks"] = False             # T=pin gpu clocks and fan, F=don't
-globalParameters["HardwareMonitor"] = True        # False: disable benchmarking client monitoring clocks using rocm-smi.
+globalParameters["HardwareMonitor"] = False        # False: disable benchmarking client monitoring clocks using rocm-smi.
```

Comment:
Is this change intended?
If so, please provide some explanation why you need to change this.

Reply:
No, this is unintentional.